rts/at/bridge/src/main/java/org/jboss/narayana/rest/bridge/inbound/InboundBridgeManager.java

```diff
@@ -45,13 +45,9 @@
 
     private final Map<String, Xid> transactionXids;
 
-    public static InboundBridgeManager getInstance() {
+    public static synchronized InboundBridgeManager getInstance() {
```

Comment:
I used synchronized since it the overhead compared with the double check idiom is small. findbugs detects that the idiom is known to be unsafe:

http://www.cs.umd.edu/~pugh/java/memoryModel/DoubleCheckedLocking.html.